salt/tests/unit/modules/files/test_metalk8s_network.yaml

```diff
@@ -284,7 +283,6 @@ get_control_plane_ingress_external_ips:
       node-1: 1.1.1.2
       node-2: 1.1.1.3
     result:
-      - 1.1.1.1
```

Comment:
Can you change this test to be sur the cp ingress ip is always the first (basically changing the `cp_ingress_ip_ret` to `1.1.1.2`